### PR TITLE
Geolocation to display

### PR DESCRIPTION
**Pull request by jasulcaf**
I have written up a small page where we can see the Geolocation data. This should reflect the longitude and latitude of wherever a User is located. The function of Watching and Stopwatching will allow for us to continuously see if user switches position and update accordingly.

Note: It is already dark outside when I'm writing this, so I can't test this outside. I will want to do further tests to make sure that the longitude and latitude are correctly grabbing while players are continuously on the move.

### CODE REVIEW

**Review by nd1az — APPROVED** [review]
Tested the code on ns preview and the module works perfectly fine, my location kept getting updated and changed as I walked around.

**Review by gypark23 — CHANGES_REQUESTED** [review]
I think the merge conflicts come from our most recent PR #84 . Not sure what you edited in the five files but if you didn't modify them then don't overwrite the changes.

**Comment by jasulcaf** [comment]
Waiting until checks have been passed - since I have merged Noah's PR, all the tests should be passing

**Review by Strongbrother1 — APPROVED** [review]

**Pull request merged**
